ui/app/registration/controllers/patientCommonController.js

```diff
@@ -39,6 +45,49 @@ angular.module('bahmni.registration')
                 }
             });
 
+            $scope.checkDuplicatePatients = function () {
+                var patientGivenName = $scope.patient.givenName || '';
+                var patientLastName = $scope.patient.familyName || '';
+                var gender = $scope.patient.gender || '';
+                var birthDate = $scope.patient.birthdate || '';
+                var contactNumber = $scope.patient.PERSON_ATTRIBUTE_TYPE_PHONE_NUMBER || '';
+                if (birthDate != '') {
+                    birthDate = new Date(birthDate);
+                    birthDate = convertDate(birthDate);
```

Comment:
@IipperMangesh the code on line 56 overrides the result of line 55 (i.e. we'll need to keep one of the 2 lines above)

Reply:
Yes, Added into one line code.